src/Uno.UWP/UI/ViewManagement/ApplicationView.Android.cs

```diff
@@ -39,6 +39,28 @@ public bool IsScreenCaptureEnabled
 			}
 		}
 
+		public string Title
+		{
+			get
+			{
+				if (!(ContextHelper.Current is Activity activity))
+				{
+					throw new InvalidOperationException($"{nameof(Title)} API must be called when Activity is created");
+				}
```

Comment:
This check could be extracted into a helper method.

Reply:
Will do 👍